deepspeed/.github/workflows/amd-mi200.yml

```diff
@@ -0,0 +1,86 @@
+name: amd-mi200
```

Comment:
@eonsparks - something looks to be weird merge-artifact wise here, since this file should already be in the repo, same with the nv-mii yml file.  Could you take a look at this or synchronize your branch with the latest in DeepSpeed?